components/aws/sagemaker/common/_utils.py

```diff
@@ -65,7 +66,10 @@ def add_default_client_arguments(parser):
 
 def get_sagemaker_client(region, endpoint_url=None):
     """Builds a client to the AWS SageMaker API."""
-    client = boto3.client('sagemaker', region_name=region, endpoint_url=endpoint_url)
+    session_config = botocore.config.Config(
+        user_agent='sagemaker-on-kubeflow-pipelines-v0.2'
```

Comment:
Do you plan to update the version for every release?  is there more elegant way like you read from version file to build user agent?

Reply:
updated to fetch at runtime